packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx

```diff
@@ -220,6 +213,7 @@ export const TransactionReviewModalContent = ({
         stakeType,
         shouldCheckTxTimeValidity,
         isInternalTransfer,
+        isTradingOrSell: !!precomposedForm.activeTradingSection,
```

Comment:
```suggestion
        isTrading !!precomposedForm.activeTradingSection,
```

trading is sell, buy swap, so no need to call it like this 😀

Reply:
done